services/QuillLMS/lib/tasks/users.rake

```diff
@@ -39,4 +39,19 @@ namespace :users do
       admin_user.update(role: User::ADMIN)
     end
   end
+
+  task mark_google_clever_admins_verified: :environment do
+    User.left_outer_joins(:user_email_verification)
+      .where.not(clever_id: nil)
+      .or(User.where.not(google_id: nil)) # note that the position of `or` matters a lot when you have multiple `where` clauses
+      .where(role: User::ADMIN)
+      # I was going to use the open-ended `..` here, but Rubocop apparently can't handle a `..` without a value at the end and chokes.  `..nil` is documented as being equivalent to an unterminated `..`
+      .where(created_at: DateTime.new(2023, 2, 27)..nil) # this is the date we launched self-service admin sign-up
```

Comment:
I think this probably works, but there could theoretically be some admins that were invited by an existing admin (bypassing the admin flow and therefore email verification) who signed up in that timeframe, so it might be safer to check for the existence of an `approval_status` value to be sure that they really did use the self-service flow.

Reply:
Good call.  Per our Zoom conversation, I'll update the logic here.